nifty-renderer-libgdx/src/main/java/de/lessvoid/nifty/gdx/render/GdxGL.java

```diff
@@ -1,10 +1,11 @@
 package de.lessvoid.nifty.gdx.render;
 
 import com.badlogic.gdx.Gdx;
-import com.badlogic.gdx.graphics.GL10;
-import com.badlogic.gdx.graphics.GL11;
 
+import com.badlogic.gdx.utils.BufferUtils;
 import de.lessvoid.nifty.render.batch.spi.GL;
+import org.lwjgl.opengl.GL11;
```

Comment:
@Sythelux It's OK if you have to direct import APIs that are not available in the LibGDX GL API. There a bunch of direct references in there, however, that are in the `Gdx.gl` interface, so I would like those to use the GDX interface rather than LWJGL directly.

Let me know once you have finished doing that (shouldn't take too long, I don't think) and I'll merge your changes.